tests/test_OMSessionCmd.py

```diff
@@ -3,7 +3,7 @@
 
 def test_isPackage():
     omczmq = OMPython.OMCSessionZMQ()
```

Comment:
Why is this test still using `OMCSessionZMQ`?
I understand that `OMCSessionZMQ` is still in code with depreciation warning but the test should use the latest code.